test/support/api.ex

```diff
@@ -30,7 +30,7 @@ defmodule MongoosePush.Support.API do
 
   def get(path) do
     %Response{status_code: status, headers: headers, body: body} =
-      HTTPoison.get!("https://localhost:8443" <> path)
+      HTTPoison.get!("https://localhost:8443" <> path, [], hackney: [:insecure])
```

Comment:
Just out of curiosity, why's that necessary?

Reply:
It's necessary because of `bad certificate` problems. We passed this option in `MongoosePush.Support.API.post` and not this one because it wasn't used anywhere in the tests anywhere so far.